src/assets/css/main.css

```diff
@@ -434,3 +444,8 @@ br.big {
 em-emoji-picker {
   width: 100%;
 }
+
```

Comment:
Run prettier. Or make sure your husky gets set up correctly by running yarn lint, because for some reason it didnt.